assets/src/edit-story/components/panels/panel/shared/title.js

```diff
@@ -48,7 +48,7 @@ function getBackgroundColor(isPrimary, isSecondary, theme) {
   return 'transparent';
 }
 
-const Header = styled.h2`
+const Header = styled.h2.attrs({ role: 'button' })`
```

Comment:
I just want to say, that this seems extremely _unsemantic_ to me and want it to be reflected on the record, that I don't like it.

But if the a11y-team wants this, I'll approve it.

Oh, and this single line summarizes the _unsemanticness_ of this PR perfectly per my opinion.

Reply:
Acknowledged.